packages/core/src/components/Form.tsx

```diff
@@ -350,6 +352,7 @@ export default class Form<
       props.idPrefix,
       props.idSeparator
     );
+
     const nextState: FormState<T, S, F> = {
```

Comment:
Let's just restore this back to what it was
```suggestion
    );
    const nextState: FormState<T, S, F> = {
```